content/references/http-websocket-apis/public-api-methods/server-info-methods/server_info.md

```diff
@@ -40,13 +51,14 @@ An example of the request format:
 ```sh
 #Syntax: server_info
 rippled server_info
+rippled server_info counters # for optional performance statistics
 ```
 
 <!-- MULTICODE_BLOCK_END -->
 
 [Try it! >](websocket-api-tool.html#server_info)
 
-The request does not take any parameters.
+The request takes in an optional `counters` parameter. It will render metrics pertaining to Job Queue, nodestore and RPC call counters.
```

Comment:
suggest "metrics including those that pertain to ". Otherwise, the language makes the following seem complete instead of a subset.